### DISCUSSION TITLE
Integration tests with jest

### DISCUSSION THREAD

**Discussion by romainZzZzZ** [nartc/mapper#511] [Q&A]
Hello, I have a repo setup with NX (14.0.5) and jest (27.2.3). I try to make integration tests with the mapping but strangely the mapped objects are empty. I've tried to check the configuration of the library by comparing it to your integration tests in your repo. I mostly lacked the import 'reflect-metadata'; but I guess it's only for POJOs mapping.

Have you had a hard time configuring your repo to work ?

**Comment by zgid123** [comment]
@romainZiba can you share your config for your jest test and the usage?
